underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -218,6 +225,54 @@ public boolean exists(String path) throws IOException {
     return hdfs.exists(new Path(path));
   }
 
+  @Override
+  public AccessControlList getAcl(String path) throws IOException {
+    FileSystem hdfs = getFs();
+    // TODO(chen): for hadoop <= 2.3, ACL is not supported, there is no getAclStatus method.
```

Comment:
Will this break the release script which builds Alluxio for all Hadoop versions?

Reply:
@ohboring This was used before: https://github.com/Alluxio/alluxio/blob/master/underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java#L299

but, I don't know how to add additional versions.